src/test/dialogs.test.js

```diff
@@ -0,0 +1,209 @@
+var expect = require("chai").expect;
+var React = require("react/addons");
+var TestUtils = React.addons.TestUtils;
+
+var DialogActions = require("../js/actions/DialogActions");
+var DialogEvents = require("../js/events/DialogEvents");
+var DialogStore = require("../js/stores/DialogStore");
+
+var expectAsync = require("./helpers/expectAsync");
+
+describe("Dialog store", function () {
+
+  describe("alert", function () {
+
+    it("show sends correct message", function (done) {
+      DialogActions.alert("test alert");
```

Comment:
Shouldn't we invert the order with the `.once` here? I mean logically speaking I'd like to bind first and trigger the action second, just like we do in all our tests?